composer.json

```diff
@@ -19,7 +19,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6",
```

Comment:
require will handle those. we would love to use this lib with php5.4.

composer will calculate the actual requirements based on the php-version requirement of your project in combination with all dependencies.
In other words you define what you need, dependencies what they need. you should not take your dependencies requirements here.